src/models/constructor-items/common.ts

```diff
@@ -178,6 +178,7 @@ export interface ButtonProps {
     metrikaGoals?: MetrikaGoal;
     pixelEvents?: ButtonPixel;
     target?: string;
+    selected?: boolean;
```

Comment:
Why do we need to add this?

Reply:
Sorry, did not removed after testing one possible solution. Done.